infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -57,6 +57,7 @@ maketemp() {
 # find_files is a helper to exclude .git directories and match only regular
 # files to avoid double-processing symlinks.
 find_files() {
+  EXCLUDE_LINT_DIR=${EXCLUDE_LINT_DIR:-}
```

Comment:
@milesmatthias Instead of only allowing a single directory, could we accept a list of directories?